admin/company/settings/sso-configuration.md

```diff
@@ -12,8 +12,8 @@ Follow the steps on this page to configure SSO for your company.
 
 1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
 2. In the left navigation, select your company in the drop-down menu.
-3. Select **Settings**.
-4. Select **Add Domain** and continue with the on-screen instructions to add the TXT Record Value to your domain name system (DNS).
+3. Select **Domain management**.
+4. Select **Add a domain** and continue with the on-screen instructions to add the TXT Record Value to your domain name system (DNS).
```

Comment:
Same as [this comment](https://github.com/docker/docs/pull/17705/files#r1256288159), something to be aware of.